### ISSUE TITLE
To-do (eventually): Clean up unused strings, code, etc

### ISSUE THREAD

**Issue by BuffelSaft** [rh-hideout/pokeemerald-expansion#2435]
I've noticed a handful of things in the battle engine that are completely unused. e.g. there are strings specific to Flame Orb and Toxic Orb, but neither are used when the orbs activate. There are also a handful of battle scripts and battel scripting commands that aren't were replaced without deleting the old versions.

Wold be good to clean all this up one day, once we've run out of actual issues.

**Comment by pkmnsnfrn** [comment]
I agree that we should do this, but this issue isn't particularly actionable. If there was a checklist or something to chip away at, it would be easier to get hands working on this.

Frankfurter has also been doing an excellent job at this naturally. Going to close for now.
